src/components/Dashboard/TodoList.vue

```diff
@@ -458,7 +484,7 @@ export default {
   },
 
   computed: {
-    ...mapGetters(['isBComp', 'isCoop', 'isRoleStaff']),
+    ...mapGetters(['getIncorporationNumber', 'isBComp', 'isCoop', 'isRoleStaff']),
```

Comment:
Incorporation Number is the same as Entity Inc No. Let's stick with a single property :)

(Add a comment where you use this if it helps clarity.)